docker-compose.yml

```diff
@@ -21,6 +21,16 @@ x-gunicorn-master: &x-gunicorn-master
     - SQL_HOST=database
     - SQL_USER=postgres
     - SQL_DATABASE=postgres
+    - EMAIL_PROVIDER=AWS
+    - EMAIL_ADDRESS=noreply@ractf.co.uk
+    - "EMAIL_NAME=2021 - RACTF"
+    - AWS_FILES_BUCKET_NAME=files-2021.ractf.co.uk
+    - AWS_FILES_BUCKET_DOMAIN=files-2021.ractf.co.uk
+    - USE_AWS_S3_FILE_STORAGE=true
+    - AWS_DEFAULT_REGION=eu-west-2
+    - AWS_ACCESS_KEY_ID=AKIAZFVPOYEPJLOZ6G6K
+    - AWS_SECRET_ACCESS_KEY=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
```

Comment:
You fucking idiot

Reply:
I am indeed a complete tit, these keys have a source IP restriction however and I have just rotated them